tests/test_split.py

```diff
@@ -258,11 +270,32 @@ def do_query(
                     ],
                     "limit": 10,
                 },
-                TableSource("events", ColumnSet([])),
+                TableSource("events", column_set),
             )
         ),
         False,
     ),  # Valid query but not enough columns to split.
+    (
+        "event_id",
+        "project_id",
+        "timestamp",
+        ClickhouseQuery(
+            LogicalQuery(
+                {
+                    "selected_columns": [["f", ["event_id"], "not_event_id"]],
```

Comment:
I'm guessing this doesn't get up to the point where we actually run validate_aliases() since we aren't attempting to select any more than the minimum columns.

Reply:
You are right. Will fix.